packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -104,17 +104,14 @@ export class GCFBootstrapper {
         request.get('x-github-delivery') ||
         request.get('X-GitHub-Delivery') ||
         '';
-      const delivery =
-        request.get('x-hub-signature') || request.get('X-Hub-Signature') || '';
+      // TODO: add test to validate this signature is used on the initial
+      // webhook from GitHub:
       const signature =
-        request.get('x-github-delivery') ||
-        request.get('X-GitHub-Delivery') ||
-        '';
+        request.get('x-hub-signature') || request.get('X-Hub-Signature') || '';
```

Comment:
I'm not sure this is correct. Consider these lines
https://github.com/probot/serverless-gcf/blob/master/index.js#L39-L50. I believe the "signature" does need to be "x-github-delivery" and we just named things poorly.

Reply:
works for me, I will address.

Reply:
@orthros okay, reverted the header, @sofisl is testing everything out.